lib/src/facts/solaris/virtualization_resolver.cc

```diff
@@ -32,23 +32,7 @@ namespace facter { namespace facts { namespace solaris {
             return vm::zone;
         }
 
-        string guest_of;
-
-        // Use the same timeout as in Facter 2.x
-        const uint32_t timeout = 20;
-        try {
-            each_line(
-                "/usr/sbin/prtdiag",
```

Comment:
The solaris [dmi resolver](https://github.com/puppetlabs/facter/blob/3.6.x/lib/src/facts/solaris/dmi_resolver.cc#L46) also collects prtdiag information, including product name, so I think it's okay to remove this.

Reply:
:+1: to not calling out to the same command twice. Good catch!